src/helpers.js

```diff
@@ -176,7 +176,7 @@ export function arrayEquals(arr1, arr2) {
 }
 
 export function logDebug() {
-    if (window.debugging === "all" || window.debugging === "machines")
+    if (window.debugging === "all" || window.debugging?.includes("machines"))
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1289-20231107-105633-1b57fd30-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)